web/tests/proportion-field-service-spec.js

```diff
@@ -0,0 +1,141 @@
+describe('proportion-field service', function() {
+    beforeEach(function () {
+
+        angular.mock.module('incuna-surveys.proportion-field');
+
+        inject(function(proportionField) {
+            this.proportionField = proportionField;
+        });
+
+        this.fields = [
```

Comment:
Maybe it is confusing that the function is destructively modifying the `fields` object.